lib/cli.js

```diff
@@ -877,6 +877,7 @@ exports.search = function(keywords) {
 
 exports.init = function(options) {
   if (!shell.which('git')) return console.log(chalk.red('    Prerequisite not installed: git'));
+  if (!shell.which('mongod')) return console.log(chalk.red('    Prerequisite not installed: mongod'));
```

Comment:
Is it 'mongod' on all platforms? Most concerned about windows, where perhaps `mongod.exe` instead? Ran into a similar problem with `npm`.